src/ck-ldb/LBManager.h

```diff
@@ -99,15 +101,24 @@ class CkLBOptions
 {
  private:
   int seqno;  // for centralized lb, the seqno
-  const char* legacyName;
+  std::string legacyName;
  public:
-  CkLBOptions() : seqno(-1), legacyName(nullptr) {}
-  CkLBOptions(int s) : seqno(s), legacyName(nullptr) {}
+  CkLBOptions() : seqno(-1), legacyName() {}
+  CkLBOptions(int s) : seqno(s), legacyName() {}
   CkLBOptions(int s, const char* legacyName) : seqno(s), legacyName(legacyName) {}
```

Comment:
CkLBOptions(int s, const char* legacyName) : seqno(s), legacyName(legacyName) {}

should be something like this:

CkLBOptions(int s, const char* n) : seqno(s), legacyName(n ? n : "") {}

Reply:
Thanks -- good catch. I fixed this and rebased.